src/Build/Resources/Strings.resx

```diff
@@ -2095,11 +2095,7 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
       LOCALIZATION: {0} is integer number denoting number of bytes. 'int.MaxValue' should not be translated.
     </comment>
   </data>
-  <!--
```

Comment:
do we need to preserve this comment?

Reply:
I guess we do. I edited this file with VS editor and it probably removed it + I missed that change. Fixed.

Reply:
Unfortunately this file must be edited manually - otherwise not only comments, but as well some functional data (like conditions) are lost

Reply:
I reverted all changes and added manually only what I wanted to add :)